Lib/test/test_bytes.py

```diff
@@ -962,6 +962,11 @@ def test_translate(self):
         c = b.translate(None, delete=b'e')
         self.assertEqual(c, b'hllo')
 
+    def test_sq_item(self):
```

Comment:
Can you add some tests that call sequence_getitem with a non-negative index as well?

Reply:
I meant, could you add a test that calls sequence_getitem successfully? (to make sure the successful behavior is not impacted)

Reply:
I meant, could you add a test that calls sequence_getitem successfully? (to make sure the successful behavior is not impacted)